scripts/utils/upload_and_generate_geojson.py

```diff
@@ -105,22 +94,22 @@
     # This dataset has quite some dimensions, so if we would parse all information the end-user
     # would be overwhelmed by all options. So for the stac items that we generate for the frontend
     # visualizations a subset of the data is selected. Of course, this operation is dataset specific.
-    for k, v in MAP_SELECTION_DIMS.items():
-        if k in ds.dims and ds.coords:
-            ds = ds.sel({k: v})
-        else:
-            try:
-                # assume that coordinates with strings always have same dim name but with n
-                ds = ds.sel({"n" + k: k == v})
-            except:
-                raise ValueError(f"Cannot find {k}")
-
-    dimvals = get_dimension_values(ds, dimensions_to_ignore=DIMENSIONS_TO_IGNORE)
-    dimcombs = get_dimension_dot_product(dimvals)
+    # for k, v in MAP_SELECTION_DIMS.items():
+    #     if k in ds.dims and ds.coords:
+    #         ds = ds.sel({k: v})
+    #     else:
+    #         try:
+    #             # assume that coordinates with strings always have same dim name but with n
```

Comment:
Why is this commented out? In principle it should go over this correctly

Reply:
Dataset only has one single dimension, no need to select certain ones or make combinations.

Reply:
If the parameters are filled in correcly, there is no need to comment this out. It should pass over without any hassle. Not sure if its correctly formatted now.. Lets have a call about it.